services/user/Supervisor/ui/src/supervisor.ts

```diff
@@ -60,15 +60,17 @@ export class Supervisor implements AppInterface {
             "Redundant setting parent origination",
         );
 
-        if (callerOrigin === supervisorDomain) {
-            console.info(
-                "TODO: handle calls directly from the rootdomain itself",
-            );
+        if (callerOrigin === siblingUrl(null, null, null, true)) {
```

Comment:
@James-Mart

Reply:
I removed my origin rewrite and left the app explicitly set to homepage